tests/commands/completion/fixtures/command_with_colons.py

```diff
@@ -1,12 +1,10 @@
 from __future__ import annotations
 
 from cleo.commands.command import Command
+from cleo.helpers import option
 
 
 class CommandWithColons(Command):
-    """
-    Test.
-
-    command:with:colons
-        { --goodbye }
-    """
+    name = "command:with:colons"
+    options = [option("goodbye", flag=False)]
```

Comment:
Not that it likely matters, but isn't this a flag?

```suggestion
    options = [option("goodbye")]
```